pkg/scheduler/apis/config/v1alpha2/zz_generated.conversion.go

```diff
@@ -210,7 +209,8 @@ func autoConvert_v1alpha2_KubeSchedulerConfiguration_To_config_KubeSchedulerConf
 	if err := Convert_v1alpha2_KubeSchedulerLeaderElectionConfiguration_To_config_KubeSchedulerLeaderElectionConfiguration(&in.LeaderElection, &out.LeaderElection, s); err != nil {
 		return err
 	}
-	if err := v1alpha1.Convert_v1alpha1_ClientConnectionConfiguration_To_config_ClientConnectionConfiguration(&in.ClientConnection, &out.ClientConnection, s); err != nil {
```

Comment:
Is any of the conversion change related? That seems weird.

Reply:
This is strange. Will make this back to what it was. Happened unknowingly.